core/data_restore/src/accounts_state.rs

```diff
@@ -189,136 +189,239 @@ impl FranklinAccountsState {
-        // let decoded3 = hex::decode(data3).expect("Decoding failed");
-        // let ops3 = FranklinOpsBlock::get_franklin_ops_from_data(&decoded3)
-        //     .expect("cant get ops from data");
-        // println!("ops3 {:?} \n", ops3);
-        // let block3 = FranklinOpsBlock {
-        //     block_num: 3,
-        //     ops: ops3,
-        // };
-
         let mut tree = FranklinAccountsState::new();
-        let updates1 = tree
-            .update_accounts_states_from_ops_block(&block1)
+        tree.update_accounts_states_from_ops_block(&block1)
             .expect("Cant update state from block 1");
-        println!("updates 1 {:?} \n", updates1);
-        println!("root hash 1 {:?} \n", tree.root_hash());
-        println!("accounts 1 {:?} \n", tree.get_accounts());
-        let updates2 = tree
-            .update_accounts_states_from_ops_block(&block2)
+        tree.update_accounts_states_from_ops_block(&block2)
             .expect("Cant update state from block 2");
-        println!("updates 2 {:?} \n", updates2);
-        println!("root hash 2 {:?} \n", tree.root_hash());
-        println!("accounts 2 {:?} \n", tree.get_accounts());
-        let updates3 = tree
-            .update_accounts_states_from_ops_block(&block3)
+        tree.update_accounts_states_from_ops_block(&block3)
             .expect("Cant update state from block 3");
-        println!("updates 3 {:?} \n", updates3);
-        println!("root hash 3 {:?} \n", tree.root_hash());
-        println!("accounts 3 {:?} \n", tree.get_accounts());
-        let updates4 = tree
-            .update_accounts_states_from_ops_block(&block4)
+        tree.update_accounts_states_from_ops_block(&block4)
             .expect("Cant update state from block 4");
-        println!("updates 4 {:?} \n", updates4);
-        println!("root hash 4 {:?} \n", tree.root_hash());
-        println!("accounts 4 {:?} \n", tree.get_accounts());
-        let updates5 = tree
-            .update_accounts_states_from_ops_block(&block5)
-            .expect("Cant update state from block 4");
-        println!("updates 5 {:?} \n", updates5);
-        println!("root hash 5 {:?} \n", tree.root_hash());
-        println!("accounts 5 {:?} \n", tree.get_accounts());
+        tree.update_accounts_states_from_ops_block(&block5)
+            .expect("Cant update state from block 5");
 
         assert_eq!(
             tree.root_hash().to_string(),
-            "Fr(0x0f220069602ed8f8c4557fdde71baf5220bbf237790adf67f49280b84588acf2)".to_string()
+            "Fr(0x06210ec05cbdd8a974447f23aa9a2e2a6c6fe500620bd4ffc0d9752d1034a062)".to_string()
         );
     }
 
     #[test]
-    // TODO: Fix test of tree restore
-    #[ignore]
     fn test_tree_inconsistent_update() {
```

Comment:
What outcome of this test is expected? If it is "inconsistent update" then why this test should pass?